images/bigquery/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN apt-get update && apt-get install -y \
 
 ARG CLOUD_SDK_VERSION=390.0.0
 ARG BIGQUERY_LIBRARY_VERSION=0.26.0
-ARG RUAMEL_VERSION=0.16
+ARG RUAMEL_VERSION=0.16.5
 
 RUN pip3 install --no-cache-dir google-cloud-bigquery==${BIGQUERY_LIBRARY_VERSION} ruamel.yaml==${RUAMEL_VERSION}}
```

Comment:
might need to upgrade pip itself first.

https://github.com/kubernetes/test-infra/blob/b362e57f79b0eb9424e988001645838e170104de/kettle/Dockerfile#L33-L34